### DISCUSSION TITLE
关于“自动公招”的优化

### DISCUSSION THREAD

**Discussion by Xiafeng-maples** [MaaAssistantArknights/MaaAssistantArknights#11312] [Ideas]
目前自动公招只能选自动使用加急来完成快速刷新
考虑到其实有的人许可票远多于加急，能不能加一个选项，当识别全3星后自动选择1小时招募，然后取消，以此取代加急刷新，识别到高级tag后再正常招募，加急。这样也给许可多的人一个选择

**Comment by ABA2396** [comment]
这玩意之前讨论过很多次了，这样的操作纯粹就是浪费。许可的获取量是很有限的，平均下来每天就3.7张，也就是跑一次任务用4张，基本就持平了。就算你信用商店刷出来的票子很多，那你抽空用一次加急或者一天两次任务也完全够用了

**Reply by lucienshawls** [reply]
首先，选择1小时招募会亏绿票，因为此时可能会刷出二星干员；其次，即使不选择任何tag，公招也有概率刷出四星甚至五星干员，这个概率随着招募时长的提升而提升。

因此，除非你要刷小车，否则这样的操作一定是亏的；至于你所提出的“**以亏绿票为代价，迅速将大量公招券转换为绿票**”这一需求，我不知道它是否足够普遍，以至于需要开发者去实现。
